packages/frontend/src/components/DashboardFilter/index.tsx

```diff
@@ -31,9 +35,13 @@ const DashboardFilter: FC = () => {
                 onInteraction={setIsOpen}
                 position="bottom-right"
                 lazy={false}
-                disabled={isLoading}
+                disabled={isLoading || !hasCharts}
```

Comment:
We already have access to the `dashboard` via the context so we can avoid prop drilling.